workbench/palette-comparison/style.css

```
@@ -0,0 +1,339 @@
+}
+
+.palette-values i {
+  width: 0.85rem;
+  height: 0.85rem;
+  border: 1px solid var(--workbench-line);
+  border-radius: 50%;
+  background: var(--chip);
+}
+
+.palette-values code {
+  font-family: ui-monospace, "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)